src/main/java/core/basesyntax/services/FileReaderImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.services;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String filePath) {
+        try (BufferedReader reader = new BufferedReader(new java.io.FileReader(filePath))) {

```

Comment:
Please use try-with-resources to ensure that the `BufferedReader` is closed properly after use.